src/main/java/pl/project13/maven/git/log/VerboseLogger.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * This file is part of git-commit-id-plugin by Konrad 'ktoso' Malawski <konrad.malawski@java.pl>
+ *
+ * git-commit-id-plugin is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * git-commit-id-plugin is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with git-commit-id-plugin.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package pl.project13.maven.git.log;
+
+import org.apache.maven.plugin.Mojo;
+
+/**
+ * Bridges logging to standard Maven log adhering to verbosity level.
+ */
+public class VerboseLogger implements VerboseLog {
```

Comment:
How about using this interface (VerboseLog) everywhere, to not touch Mojo directly in many places?
(And pass around the VerboseLog, instead of the Mojo like it is now on this branch)